age/src/protocol.rs

```diff
@@ -63,7 +63,7 @@ impl Encryptor {
     /// recipients.
     ///
     /// Returns `None` if no recipients were provided.
-    pub fn with_recipients(recipients: Vec<Box<dyn Recipient>>) -> Option<Self> {
+    pub fn with_recipients(recipients: Vec<Box<dyn Recipient + Send>>) -> Option<Self> {
```

Comment:
This change should be documented in the changelog. Non-blocking; I can do this in the release PR.